docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Additional database version information
+{: .no_toc }
+
+{% include page-toc.md %}
+
+
+## Where can I find my database version?
+
+### Option 1: View in your list of databases
+1. Select databases on the sidebar
+2. View your database's version under the version column
+
+### Option 2: View in your database's summary
+1. Select databases on the sidebar
+2. Click on your database's name
+3. View your database's version in the summary
+
+## Why is my database's version "Unknown"?
+
+| Database Status | Explanation | 
```

Comment:
FYI, the "Explanation" header should be "Description" :) Not a biggie and I've got to write up the standards for this so don't fret over it otherwise :)